SECURITY.md

```diff
@@ -0,0 +1,19 @@
+# Security Policy
+
+## Supported Versions
+
+Qiskit.org only  supports the online version at a time both for bug, which covers `https://*.qiskit.org/*`.
```

Comment:
Small suggestion :)

```suggestion
Qiskit.org only supports the online version at a time both for bug, which covers `https://*qiskit.org/*`.
```

Reply:
hmm... my interpretation of ´*qiskit.org´ is that a website called ´fooqiskit.org´ would be covered by the policy. I guess you would like to include the canonical domain. Maybe we can have that one explicitly.

Reply:
You're right... I suggest the change because my interpretation of `*.qiskit.org` is that `https://qiskit.org/` and all the subpages from there are not covered

Reply:
Just seen it's resolved [here](https://github.com/Qiskit/qiskit.org/pull/2979#discussion_r1108437537), thank you!